Server/ProtoGenerated/ProtoGenerated.csproj

```diff
@@ -10,7 +10,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="generated" />
```

Comment:
Seems like this is actually causing problems with the build. Removing this should make the unit tests pass.